### DISCUSSION TITLE
Deploying to Vercel

### DISCUSSION THREAD

**Discussion by tom-sherman** [avisi-cloud/structurizr-site-generatr#294] [Show and tell]
Thought I'd share a demo I created of deploying a structurizr-site-generatr site using Vercel. The platform made it relatively easy to customise their build image to install Java and download the generator from GitHub releases

https://github.com/tom-sherman/vercel-structurizr-site-generatr-demo

We're using this internally at OVO to distribute our architecture and infrastructure.

**Comment by dirkgroot** [comment]
Cool, thanks for sharing!
